Ivanchuk_Pavel/02.Git.Local/README.md

```diff
@@ -0,0 +1,21 @@
+##02.GIT.Local 
+### My first experience using the GIT version control system.
+I use a virtual machine deployed in the Proxmox hypervisor on a production server.
+I use Linux Debian version 11.7 as the operating system.
+At the moment I installed git, installed zhsk instead of bash (But I have not yet figured out how it
+works and how to configure it :)). I created a new directory in which I initialized the GIT repository.
+I signed whose GIT it is, but not yet globally. I created a new file (I’m writing in it now), in which
+I wrote the first line. Staged and committed this file. I created a new thread and am writing this text.
+
+### Вс 24 сен 2023 13:04:38 +03
+I decided to also enter the date. Well, I made a second branch, committed the file with the changes 
+to it and merged it with the master branch. I checked everything and added the tag as a new version, 
+but then I realized that I added it to the master branch, and not to a separate commit. 
+I removed the tag, moved to the commit I needed and already added the tag to it.
+
+### Вс 24 сен 2023 14:02:38 +03
+I made changes to the report file, then removed them from the working directory. Moved to another branch 
+to create a new commit.
+
+I didn’t know what format the report should look like, so I wrote it like this. Next time I will write
```

Comment:
You can combine both files in your PR to one file report.

Just divide your command history by tasks part